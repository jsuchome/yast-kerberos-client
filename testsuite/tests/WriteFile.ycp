/**
 *  WriteFile.ycp
 *  Test of Kerberos::WriteFile function
 *  Author:	Jiri Suchomel <jsuchome@suse.cz>
 *  $Id$
 */

{
    // testedfiles: Kerberos.ycp
    import "Testsuite";
    import "Kerberos";

    path p = .etc.krb5_conf.v.libdefaults.default_realm;

    Testsuite::Test(``(Kerberos::WriteFile (p, "SUSE")), [ $[], $[], $[] ], 0);

    Testsuite::Test(``(Kerberos::WriteFile (p, "")), [ $[], $[], $[] ], 0);

    Testsuite::Test(``(Kerberos::WriteFile (p, nil)), [ $[], $[], $[] ], 0);
}
