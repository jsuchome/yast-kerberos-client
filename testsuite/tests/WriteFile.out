Write	.etc.krb5_conf.v.libdefaults.default_realm "SUSE" true
Return	true
Return	false
Return	false
